src/pages/EventAdministration/components/EventFormEditor.tsx

```diff
@@ -1,44 +1,138 @@
-import { EventFormCreate } from 'types';
-import { useFormById, useCreateForm, useFormSubmissions } from 'hooks/Form';
-
+import { EventFormCreate, Form } from 'types';
+import { useFormById, useCreateForm, useFormSubmissions, useFormTemplates, useDeleteForm } from 'hooks/Form';
+import { useForm } from 'react-hook-form';
+import { useSnackbar } from 'hooks/Snackbar';
 // Material UI
-import { Typography, Button } from '@mui/material';
+import { Typography, Button, styled, Stack } from '@mui/material';
 
 // Project components
+import Expand from 'components/layout/Expand';
 import FormEditor from 'components/forms/FormEditor';
+import FormView from 'components/forms/FormView';
 import { FormType, FormResourceType } from 'types/Enums';
+import { Box } from '@mui/system';
+import VerifyDialog from 'components/layout/VerifyDialog';
 
 export type EventFormEditorProps = {
   eventId: number;
   formId: string | null;
   formType: FormType;
 };
 
+export type delteFormButtonType = {
```

Comment:
```suggestion
export type DeleteFormButtonType = {
```

Liten skrivefeil og alltid stor bokstav på typer